File: abc/abc305/a.rs
// https://atcoder.jp/contests/abc305/submissions/42235536

use::proconio::input;

fn main() {
  input! {
    n: u8
  }

  if (n % 5 < 3) {
    println!("{}", n - n % 5);
  } else {
    println!("{}", n + (5 - n % 5));
  }
}

File: abc/abc305/a.cs
// https://atcoder.jp/contests/abc305/submissions/46508749

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    if(n % 5 <= 2)
    {
      Console.Write(n - n % 5);
    }
    else
    {
      Console.Write(n + (5 - n % 5));
    }
  }
}

